CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - Added ``monitor_kwargs`` parameter
 - Handle the `env_kwargs` of `render:True` under the hood for panda-gym v1 envs in `enjoy` replay to match visualzation behavior of other envs
 - Added support for python config file
+- Added ``--tags`` argument to ``train.py`` to add tags to the wandb run
```

Comment:
maybe add an example in the doc (just update the one with wandb)

Reply:
I would need the command line example in the doc as minimal as possible. So I'd rather add a note after.